src/utils/event-manager/wheel-input.js

```diff
@@ -64,6 +64,7 @@ export default class WheelInput {
     if (!this.options.enable) {
       return;
     }
+    event.preventDefault();
```

Comment:
Ok, got it. It's just wheel events that are async, correct? (Due to the `setTimeout` in there.)